doc/api/next_api_changes/deprecated_empty_add_axes.rst

```diff
@@ -0,0 +1,5 @@
+Deprecations
+````````````
+
+Calling `fig.add_axes()` with no arguments currently return None,
```

Comment:
This will try and link to a method in the library.  You need to enclose in double backticks to make a literal.

Reply:
thanks Jody